<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpPage</class>
 <widget class="QWidget" name="SignUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>824</width>
    <height>624</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>824</width>
    <height>624</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>801</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_SignUpBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>420</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>390</width>
        <height>420</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QPushButton" name="pushButton_DoSignUp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_DoSignUp:enabled{
	border-width:0;
	background-color:#0a66c2;
	border-radius:20px;
	color:#ffffff;
}
QPushButton#pushButton_DoSignUp:hover{
	border-width:0;
	background-color:#004182;
	border-radius:20px;
	color:#ffffff;
}
QPushButton#pushButton_DoSignUp:pressed{
	border-width:0;
	background-color:#004182;
	border-radius:20px;
	color:#ffffff;
}
QPushButton#pushButton_DoSignUp:focus{
	border-width:0;
	background-color:#004182;
	border-radius:20px;
	color:#ffffff;
}</string>
       </property>
       <property name="text">
        <string>同意并加入</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Password">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>350</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请牢记您的邮箱，它是您重置密码的唯一凭证。</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Name">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>已有账号？</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_BackSignIn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>370</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_BackSignIn:enabled{
	border-width:0;
	background-color:transparent;
	border-radius:20px;
	color:#0a66c2;
}
QPushButton#pushButton_BackSignIn:hover{
	border-width:0;
	background-color:#d0e8ff;
	border-radius:15px;
	color:#0a66c2;
}
QPushButton#pushButton_BackSignIn:pressed{
	border-width:0;
	background-color:#d0e8ff;
	border-radius:15px;
	color:#0a66c2;
}</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Mail">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>邮箱</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>350</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>点击“同意并加入”，即表示您同意遵守PWKeeper的相关协议</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>801</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Samsung Sharp Sans</family>
          <pointsize>48</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>PWKeeper</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>专业成就卓越</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
